All feedback was address but the following:

```
************* Module block_letter
block_letter.py:5:0: R0903: Too few public methods (1/2) (too-few-public-methods)
```

I think it hints that this could be a function instead of a class?

--- a/block_letter/block_letter.py
+++ b/block_letter/block_letter.py
@@ -1,7 +1,5 @@
 'Module responsible for block letters'
 
-from functools import reduce
-
 CHARACTER_HEIGHT = 7
 CHARACTER_WIDTH = 5
 ASCII_CHARACTER_INDEXES = {
@@ -38,25 +36,21 @@ def characters_to_block_letters(string):
     'Returns string as ASCII block letters'
 
     return '\n'.join(
-        map(
-            lambda row_index: _ascii_row_for_characters([*string], row_index),
-            range(CHARACTER_HEIGHT)
-        )
+        [_ascii_row_for_characters([*string], row_index) for row_index in range(CHARACTER_HEIGHT)]
     )
 
 def _ascii_row_for_characters(characters, row_index):
     return ' '.join(
-        map(
-            lambda character: _ascii_row_for_character(
-                character, ASCII_CHARACTER_INDEXES[character][row_index]
-            ),
-            characters
-        )
+        [
+            _ascii_row_for_character(character, ASCII_CHARACTER_INDEXES[character][row_index])
+            for character in characters
+        ]
     )
 
 def _ascii_row_for_character(character, column_indexes):
-    return reduce(
-        lambda result, column_index: result + (' ', character)[column_index in column_indexes],
-        range(CHARACTER_WIDTH),
-        ''
+    return ''.join(
+        [
+            (' ', character)[column_index in column_indexes]
+            for column_index in range(CHARACTER_WIDTH)
+        ]
     )

--- a/block_letter/block_letter.py
+++ b/block_letter/block_letter.py
@@ -2,67 +2,61 @@
 
 from functools import reduce
 
-class BlockLetter:
-    'Class which transforms a string into block letters'
+CHARACTER_HEIGHT = 7
+CHARACTER_WIDTH = 5
+ASCII_CHARACTER_INDEXES = {
+    'H': (
+        (0, 4),
+        (0, 4),
+        (0, 4),
+        (0, 1, 2, 3, 4),
+        (0, 4),
+        (0, 4),
+        (0, 4)
+    ),
+    'Q': (
+        (1, 2, 3),
+        (0, 4),
+        (0, 4),
+        (0, 4),
+        (0, 4),
+        (0, 3),
+        (1, 2, 4)
+    ),
+    'V': (
+        (0, 4),
+        (0, 4),
+        (0, 4),
+        (0, 4),
+        (0, 4),
+        (1, 3),
+        (2,)
+    )
+}
 
-    CHARACTER_HEIGHT = 7
-    CHARACTER_WIDTH = 5
-    ASCII_CHARACTER_INDEXES = {
-        'H': (
-            (0, 4),
-            (0, 4),
-            (0, 4),
-            (0, 1, 2, 3, 4),
-            (0, 4),
-            (0, 4),
-            (0, 4)
-        ),
-        'Q': (
-            (1, 2, 3),
-            (0, 4),
-            (0, 4),
-            (0, 4),
-            (0, 4),
-            (0, 3),
-            (1, 2, 4)
-        ),
-        'V': (
-            (0, 4),
-            (0, 4),
-            (0, 4),
-            (0, 4),
-            (0, 4),
-            (1, 3),
-            (2,)
+def characters_to_block_letters(string):
+    'Returns string as ASCII block letters'
+
+    return '\n'.join(
+        map(
+            lambda row_index: _ascii_row_for_characters([*string], row_index),
+            range(CHARACTER_HEIGHT)
         )
-    }
+    )
 
-    @classmethod
-    def characters(cls, string):
-        'Returns string as ASCII block letters'
-
-        return '\n'.join(
-            map(
-                lambda row_index: cls._ascii_row_for_characters([*string], row_index),
-                range(cls.CHARACTER_HEIGHT)
-            )
+def _ascii_row_for_characters(characters, row_index):
+    return ' '.join(
+        map(
+            lambda character: _ascii_row_for_character(
+                character, ASCII_CHARACTER_INDEXES[character][row_index]
+            ),
+            characters
         )
+    )
 
-    @classmethod
-    def _ascii_row_for_characters(cls, characters, row_index):
-        return ' '.join(
-            map(
-                lambda character: cls._ascii_row_for_character(
-                    character, cls.ASCII_CHARACTER_INDEXES[character][row_index]
-                ),
-                characters
-            )
-        )
-
-    @classmethod
-    def _ascii_row_for_character(cls, character, column_indexes):
-        return reduce(
-            lambda result, column_index: result + (' ', character)[column_index in column_indexes],
-            range(cls.CHARACTER_WIDTH),
-            ''
-        )
+def _ascii_row_for_character(character, column_indexes):
+    return reduce(
+        lambda result, column_index: result + (' ', character)[column_index in column_indexes],
+        range(CHARACTER_WIDTH),
+        ''
+    )

--- a/block_letter/main.py
+++ b/block_letter/main.py
@@ -1,5 +1,5 @@
 'Demo of block letter in actions'
 
-from block_letter import BlockLetter
+from block_letter import characters_to_block_letters
 
-print(BlockLetter.characters('VQH'))
+print(characters_to_block_letters('VQH'))

--- a/block_letter/test_block_letter.py
+++ b/block_letter/test_block_letter.py
@@ -1,15 +1,15 @@
-'Unit test for BlockLetter'
+'Unit tests for block_letter'
 
-import unittest
-from block_letter import BlockLetter
+from unittest import TestCase
+from block_letter import characters_to_block_letters
 
-class TestBlockLetterCharacters(unittest.TestCase):
-    'Tests for BlockLetter.characters'
+class TestCharactersToBlockLetter(TestCase):
+    'Tests for `characters_to_block_letter`'
 
     def setUp(self):
-        self.block_letter = BlockLetter
+        self.subject = characters_to_block_letters
 
-    def test_characters_with_v(self):
+    def test_characters_to_block_letters_with_v(self):
         'When the argument is is an `V`, it is expected to output the `V` block letter'
 
         expectation = (
@@ -22,9 +22,9 @@ class TestBlockLetterCharacters(unittest.TestCase):
             '  V  '
         )
 
-        self.assertEqual(self.block_letter.characters('V'), expectation)
+        self.assertEqual(self.subject('V'), expectation)
 
-    def test_characters_with_q(self):
+    def test_characters_to_block_letters_with_q(self):
         'When the argument is is an `Q`, it is expected to output the `Q` block letter'
 
         expectation = (
@@ -37,9 +37,9 @@ class TestBlockLetterCharacters(unittest.TestCase):
             ' QQ Q'
         )
 
-        self.assertEqual(self.block_letter.characters('Q'), expectation)
+        self.assertEqual(self.subject('Q'), expectation)
 
-    def test_characters_with_h(self):
+    def test_characters_to_block_letters_with_h(self):
         'When the argument is is an `H`, it is expected to output the `H` block letter'
 
         expectation = (
@@ -52,13 +52,13 @@ class TestBlockLetterCharacters(unittest.TestCase):
             'H   H'
         )
 
-        self.assertEqual(self.block_letter.characters('H'), expectation)
+        self.assertEqual(self.subject('H'), expectation)
 
-    def test_characters_with_vqh(self):
-        '''
-            When the argument is multiple characters,
-            it is expected to output characters from left-to-right
-        '''
+    def test_characters_to_block_letters_with_vqh(self):
+        (
+            'When the argument is multiple characters, '
+            'it is expected to output characters from left-to-right'
+        )
 
         expectation = (
             'V   V  QQQ  H   H\n'
@@ -70,4 +70,4 @@ class TestBlockLetterCharacters(unittest.TestCase):
             '  V    QQ Q H   H'
         )
 
-        self.assertEqual(self.block_letter.characters('VQH'), expectation)
+        self.assertEqual(self.subject('VQH'), expectation)

